test/new-e2e/tests/updater/linux_test.go

```diff
@@ -340,8 +340,10 @@ func (v *vmUpdaterSuite) TestPurgeAndInstallAPMInjector() {
 	require.NotNil(v.T(), err)
 	_, err = host.Execute(`grep "/opt/datadog-packages/datadog-apm-inject" /etc/docker/daemon.json`)
 	require.NotNil(v.T(), err)
-	_, err = host.Execute(`test -f /etc/docker/daemon.json.bak`)
+	_, err = host.Execute(`test -f /etc/docker/daemxn.json.bak`)
```

Comment:
Is this deliberate?

Reply:
Oops, good catch. That test should've been removed anyway, we're not backing up the daemon config anymore.

Reply:
Fixed in [71d47c1](https://github.com/DataDog/datadog-agent/pull/24676/commits/71d47c1410f44120d9dc522ec72ee1113e59bb3c)